WebApiMyLib/WebApiMyLib/Startup.cs

```diff
@@ -28,7 +31,18 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddSingleton<IBookRepository, MemoryRepository>();
+
+
+            services.AddDbContext<BookDbContext>(
+                options => options.UseSqlServer("Server=(localdb)\\MSSQLLocalDB;Database=MyLibrary;Trusted_Connection=True;MultipleActiveResultSets=true;"));
+            services.AddTransient<IBookRepository, BookRepository>();
+            services.AddTransient<IAutorRepository, AutorRepository>();
+            services.AddTransient<ICategoryRepository, CategoryRepository>();
+            services.AddControllers().AddNewtonsoftJson(options =>
+           options.SerializerSettings.ReferenceLoopHandling = Newtonsoft.Json.ReferenceLoopHandling.Ignore);
```

Comment:
```suggestion
            options.SerializerSettings.ReferenceLoopHandling = Newtonsoft.Json.ReferenceLoopHandling.Ignore);
```

Куда-то потерялся пробел перед options.

Reply:
исправил